packages/builder/src/utils/graphql.ts

```diff
@@ -1,10 +1,6 @@
 import { ChainId } from "common";
 
-export type GraphQLEndpoints = {
-  [key in ChainId]: string;
-};
-
-const graphQlEndpoints: GraphQLEndpoints = {
+const graphQlEndpoints: Record<ChainId, string> = {
```

Comment:
Great catch here... I guess the type was too wide and let some things slip through, whereas inferring it from the value makes it the tightest it could be.